.travis.yml

```diff
@@ -6,7 +6,7 @@ before_script:
   - curl https://raw.githubusercontent.com/jrha/krb5-travis/master/install_knc | bash
   - wget https://github.com/google/protobuf/releases/download/v3.5.1/protoc-3.5.1-linux-x86_64.zip -O /tmp/protoc.zip && cd /tmp && unzip /tmp/protoc.zip && sudo cp bin/protoc /usr/bin/ && sudo chmod ugo+x /usr/bin/protoc
   - cd /tmp && git clone https://github.com/quattor/aquilon-protocols.git && cd aquilon-protocols && git checkout upstream && ./setup.py build && sudo ./setup.py install --install-lib /usr/local/lib/aquilon/protocols/lib/python
-  - cd $TRAVIS_BUILD_DIR/tools/bootstrap_ms && python setup.py install
+  - cd $TRAVIS_BUILD_DIR/tools/bootstrap_ms && python setup.py install && ls /home/travis/virtualenv/python$TRAVIS_PYTHON_VERSION/lib/python$TRAVIS_PYTHON_VERSION/site-packages/*
```

Comment:
Another way to check this would be to try to run the Aquilon dev server outside MS with the updated config to include [unittest] section and `fake_module_location = %(srcdir)s/tests/fakemodules/`

Reply:
The kludge I've got now seems to be working, but yes it would be better to find a cleaner way to get it working.